eng/npm/README.md

```diff
@@ -1,15 +1,452 @@
+      4. You can also set this from the **refresh icon tooltip** in the Chat view, which also shows which servers will auto-start.
+         ![VS Code MCP Autostart Tooltip](https://raw.githubusercontent.com/microsoft/mcp/main/eng/npm/resources/Walkthrough/ToolTip.png)
+
+3. **Check Output for MCP server**
+         - Go to the **Output** tab in VS Code.
+         - Look for log messages confirming the server started successfully.
+
+         ![Output](https://raw.githubusercontent.com/microsoft/mcp/main/eng/npm/resources/Walkthrough/McpJsonOutput.png)
+
+You’re all set! Azure MCP Server is now ready to help you work smarter with Azure resources in VS Code.
+
+## What can you do with the Azure MCP Server?
+
+Here are some cool prompts you can try across our supported Azure services:
+
+### 🔎 Azure AI Search
+
+* "What indexes do I have in my Azure AI Search service 'mysvc'?"
+* "Let's search this index for 'my search query'"
+
+### ⚙️ Azure App Configuration
+
+* "List my App Configuration stores"
+* "Show my key-value pairs in App Config"
+
+### 📦 Azure Container Registry (ACR)
+
+* "List all my Azure Container Registries"
+* "Show me my container registries in the 'myproject' resource group"
+* "List all my Azure Container Registry repositories"
+
+### ☸️ Azure Kubernetes Service (AKS)
+
+* "List my AKS clusters in my subscription"
+* "Show me all my Azure Kubernetes Service clusters"
+
+### 📊 Azure Cosmos DB
+
+* "Show me all my Cosmos DB databases"
+* "List containers in my Cosmos DB database"
+
+### 🧮 Azure Data Explorer
+
+* "Get Azure Data Explorer databases in cluster 'mycluster'"
+* "Sample 10 rows from table 'StormEvents' in Azure Data Explorer database 'db1'"
+
+### ⚡ Azure Managed Lustre
+
+* "List the Azure Managed Lustre clusters in resource group 'my-resourcegroup'"
+* "How many IP Addresses I need to create a 128 TiB cluster of AMLFS 500?"
+
+### 📊 Azure Monitor
+
+* "Query my Log Analytics workspace"
+
+### 🔧 Azure Resource Management
+
+* "List my resource groups"
+* "List my Azure CDN endpoints"
+* "Help me build an Azure application using Node.js"
+
+### 🗄️ Azure SQL Database
+
+* "Show me details about my Azure SQL database 'mydb'"
+* "List all databases in my Azure SQL server 'myserver'"
+* "List all firewall rules for my Azure SQL server 'myserver'"
+* "List all elastic pools in my Azure SQL server 'myserver'"
+* "List Active Directory administrators for my Azure SQL server 'myserver'"
+
+### 💾 Azure Storage
+
+* "List my Azure storage accounts"
+* "Get details about my storage account 'mystorageaccount'"
+* "Create a new storage account in East US with Data Lake support"
+* "Show me the tables in my Storage account"
+* "Get details about my Storage container"
+* "Upload my file to the blob container"
+* "List paths in my Data Lake file system"
+* "List files and directories in my File Share"
+* "Send a message to my storage queue"
+
+## 🛠️ Currently Supported Tools
+
+<details>
```

Comment:
will HTML tags pose any rendering issues on NPM marketplace ?

Reply:
I don't see any issues currently with the rendering - https://www.npmjs.com/package/@azure/mcp#-%EF%B8%8F-currently-supported-tools
Currently it uses these HTML tags. I believe going forward this content will move to a shared readme template that will be used across all readmes.